src/util/defaults.ml

```diff
@@ -207,7 +207,7 @@ let _ = ()
       ; reg Experimental "exp.incremental.mode"   "'off'" "Use incremental analysis in the TD3 solver. Values: off (default), incremental (analyze based on data from a previous commit or fresh if there is none), complete (discard loaded data and start fresh)."
       ; reg Experimental "exp.incremental.stable" "true"  "Reuse the stable set and selectively destabilize it."
       ; reg Experimental "exp.incremental.wpoint" "false" "Reuse the wpoint set."
-      ; reg Experimental "exp.gcc_path"           "'/usr/bin/gcc-6'" "Location of gcc-6. Used to combine source files with cilly."
+      ; reg Experimental "exp.gcc_path"           "'/usr/bin/gcc'" "Location of gcc. Used to combine source files with cilly."
```

Comment:
Does CIL work correctly with the output of newer gcc versions?

Reply:
For the incremental test runs on the figlet repository it worked with gcc 8, 9 and 10. I don't think there are currently other use cases for combining the files with cilly other than the incremental mode. I found that getting gcc 6 to run on a newer Ubuntu system is rather cumbersome.

Reply:
Alright, let's use `gcc` then :+1: Maybe add a comment that this needs to be changed to `gcc-9` (or some other version) on OS X, where `gcc` is `clang` by default which will lead to `cilly` failing.